<launch>
  <!--
    <arg     if="$(env USB_CAMERA)" name="input_image"        default="/camera/rgb/image_raw"/>
  <arg unless="$(env USB_CAMERA)" name="input_image"      default="/camera/color/image_raw"/>
  -->
  <arg  name="input_image"  default="/camera/rgb/image_raw"/>

  <node name="clbrobot_face_detect" pkg="clbrobot_object_detect" type="clbrobot_face_detect.py" output="screen" >
      <param name="camera_topic" value="$(arg input_image)" type="string" />
  </node>
</launch>
